change/@internal-react-components-7d1c8b1b-8763-476d-800e-d4e79b33c21a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix for horizontal gallery left/right button click",
```

Comment:
For this and PR description&title, mention what the fix is. The reason for this is we us these to generate the changelog so if you view this from the lens of a developer reading the changelog - does this help me understand what has changed in this new package version? (And liekwise as a PR reviewer - what change am I expecting this PR to solve) See more examples/info here: https://github.com/Azure/communication-ui-library/blob/main/docs/references/tips-for-writing-changelog-entries.md

This could be something like:
```suggestion
  "comment": "Fix inability to click the horizontal gallery left/right button by updating zIndex.",
```

Reply:
Done, updated the PR description&title.

Reply:
Can you update this change file also?

The change file is annoying to find in vscode but you can grab the name with the copy button in the github UI here:
![image](https://user-images.githubusercontent.com/2684369/148844441-c8e974c5-9edd-4d71-a5de-986fcaeaff8c.png)
the ctrl+p in vscode and paste and enter should open the file